src/core/selection/select.ts

```diff
@@ -419,7 +419,7 @@ export class Select implements ISelect {
 			this.win.focus();
 			this.area.focus(options);
 
-			if (scrollTop && scrollParent?.scrollTo) {
+			if (scrollTop && scrollParent?.scrollTo && !options.preventScroll) {
```

Comment:
This code is written to prevent scrolling on focus. 

Please write the code of your page, how to repeat the error.

`preventScroll: true` is used in the method itself `this.area.focus(options);`

Reply:
@xdan Please try the following:
```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" class="flexbox generatedcontent">
        <head>
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jodit/3.11.2/jodit.es2018.min.css"/>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jodit/3.11.2/jodit.es2018.min.js"></script>
        <title> Test </title>
        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/prototype/1.7.3.0/prototype.js"></script>
       
        </head>
        <body id="docBody">
            <div id="fullContainer" >
                        <div style="height: 1200px; background-color: grey"> </div>
                <br />
                <textarea id="editor"></textarea>
                        <script type="text/javascript">
                                var editor = new Jodit('#editor');
                        </script>
       </body>
</html>
```

It appears there is something with the Prototype.js library that causes an issue. However, making the change to preventScroll also resolves this issue.

Reply:
Yes, this is because of prototype.js, it overrides the scrollTo method on the element and does not implement it correctly.
I don't think Jodit needs to be changed for this incorrect behavior in third library.
Try yourself on your page:
```js
document.body.scrollTo(0, 1000) // works incorrect
```

Reply:
Actually, that should be `window.scrollTo(0,0);`

Reply:
But I found a workaround. If I add `<p></p>` into the editor, the behavior of the editor works.